### ISSUE TITLE
Does not display on external screen

### ISSUE THREAD

**Issue by bensuffolk** [JohnCoates/Aerial#1301]
#### General troubleshooting tips

Before logging an issue please check that:
- [X ] You have the latest version installed (There may be a beta version that fixes your issue), see here for the latest releases and bug fixes: https://github.com/JohnCoates/Aerial/releases
- [ X] Your issue isn't already mentioned in our [issues](https://github.com/JohnCoates/Aerial/issues). You may find a workaround there or a similar request already made.
- [ X] Your problem isn't mentioned in the [troubleshooting page](https://github.com/JohnCoates/Aerial/blob/master/Documentation/Troubleshooting.md).

If none of this fixes your issue, tell us about the problem you are experiencing or the feature you'd like to request.

#### Required information
In order to help us sort your issue, we ask that you provide the following information:
- [ Macbook Pro 15" 2018] Mac model: 
- [13.3.1 (a) ] macOS version: 
- [ External screen extended with internal screen] Monitor setup: 

If appropriate, please enable `Debug mode` and `Log to disk` in `Advanced` tab and replicate your bug, then attach the `AerialLog.txt` file (You can access this file through the Advanced tab).

#### Description of issue / Feature request

Video only plays on internal screen. I have it set to use both screens and independent. I have tried setting it to just play on the external screen, and it still plays on the internal one.

This was not a problem a while ago, it has happened since I upgraded, I can not remember what version I was on before I upgarded.

**Comment by bensuffolk** [comment]
```
2023-05-23 19:38:14.143 : avInit .saver (0.0, 0.0, 1920.0, 1080.0) p: false o: false
2023-05-23 19:38:14.145 : Best match appearance : Optional(__C.NSAppearanceName(_rawValue: NSAppearanceNameAqua))
2023-05-23 19:38:14.146 : Effective Appearence : <NSCompositeAppearance: 0x7fd24c807da0>
2023-05-23 19:38:14.146 : Time Management initialized
2023-05-23 19:38:14.146 : Starting Location initialization
2023-05-23 19:38:14.147 : Location services enabled
2023-05-23 19:38:14.147 : <AerialView: 0x7fd24cc34750> AerialView setup init (V3.2.5) preview: false
2023-05-23 19:38:14.147 : Running Version 13.3.1 (a) (Build 22E772610a)
2023-05-23 19:38:14.148 : Manifest opened, going to parsing
2023-05-23 19:38:14.148 : Parsed From Joshua Michaels & Hal Bergman
2023-05-23 19:38:14.148 : Manifest opened, going to parsing
2023-05-23 19:38:14.149 : Parsed Robin Fourcade's drone shots
2023-05-23 19:38:14.149 : tvOS 16 is enabled, cached and up to date
2023-05-23 19:38:14.150 : From Joshua Michaels & Hal Bergman is enabled, cached and up to date
2023-05-23 19:38:14.150 : Robin Fourcade's drone shots is enabled, cached and up to date
2023-05-23 19:38:14.221 : Display Detection initialized
2023-05-23 19:38:14.222 : ***Display Detection***
2023-05-23 19:38:14.222 : 2 display(s) detected
2023-05-23 19:38:14.223 : 1cm = 32.28346505177813 points
2023-05-23 19:38:14.223 : npass: dict [__C.NSDeviceDescriptionKey(_rawValue: NSDeviceBitsPerSample): 8, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceColorSpaceName): NSCalibratedRGBColorSpace, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceIsScreen): YES, __C.NSDeviceDescriptionKey(_rawValue: NSScreenNumber): 1127224743, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceSize): NSSize: {1920, 1080}, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceResolution): NSSize: {72, 72}]
2023-05-23 19:38:14.224 :        bottomLeftFrame (0.0, 0.0, 1920.0, 1080.0)
2023-05-23 19:38:14.224 : npass: dict [__C.NSDeviceDescriptionKey(_rawValue: NSScreenNumber): 69734406, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceIsScreen): YES, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceSize): NSSize: {1440, 900}, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceResolution): NSSize: {144, 144}, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceColorSpaceName): NSCalibratedRGBColorSpace, __C.NSDeviceDescriptionKey(_rawValue: NSDeviceBitsPerSample): 8]
2023-05-23 19:38:14.225 :        bottomLeftFrame (-1440.0, 0.0, 1440.0, 900.0)
2023-05-23 19:38:14.225 : src orig : (0.0, 0.0)
2023-05-23 19:38:14.226 : left 1.0 below 0.0
2023-05-23 19:38:14.226 : src orig : (-1440.0, 0.0)
2023-05-23 19:38:14.227 : left 0.0 below 0.0
2023-05-23 19:38:14.227 : [id=1127224743, width=1920, height=1080, bottomLeftFrame=(0.0, 0.0, 1920.0, 1080.0), topRightCorner=(1920.0, 1080.0), isMain=true, backingScaleFactor=1.0]
2023-05-23 19:38:14.227 : [id=69734406, width=1440, height=900, bottomLeftFrame=(-1440.0, 0.0, 1440.0, 900.0), topRightCorner=(0.0, 900.0), isMain=false, backingScaleFactor=2.0]
2023-05-23 19:38:14.228 : (-1440.0, 0.0, 3360.0, 1080.0)
2023-05-23 19:38:14.228 : ***Display Detection Done***
2023-05-23 19:38:14.229 : Real screen count : 2
2023-05-23 19:38:14.229 : Using : Optional([id=1127224743, width=1920, height=1080, bottomLeftFrame=(0.0, 0.0, 1920.0, 1080.0), topRightCorner=(1920.0, 1080.0), isMain=true, backingScaleFactor=1.0])
2023-05-23 19:38:14.230 : <AerialView: 0x7fd24cc34750> setting up player layer with bounds/frame: (0.0, 0.0, 1920.0, 1080.0) / (0.0, 0.0, 1920.0, 1080.0)
2023-05-23 19:38:14.232 : allDisplays
2023-05-23 19:38:14.232 : x86_64
2023-05-23 19:38:14.233 : Using dot workaround for video driver corruption
2023-05-23 19:38:14.233 : <AerialView: 0x7fd24cc34750> backing change 2.0 isDisabled: false frame: (0.0, 0.0, 1920.0, 1080.0) preview: false
2023-05-23 19:38:14.234 : LMauth status change : 0
2023-05-23 19:38:14.241 : Refreshing video list
2023-05-23 19:38:14.242 : Poi Strings Provider initialized
2023-05-23 19:38:14.242 : Preferred languages : ["en-GB"]
2023-05-23 19:38:14.243 : Best matched language : en_GB
2023-05-23 19:38:14.243 : Defaulting to bundle
2023-05-23 19:38:14.244 : path : /Users/suffo_b/Library/Screen Savers/Aerial.saver/Contents/Resources/en.json
2023-05-23 19:38:14.244 : Community JSON : 124 entries
2023-05-23 19:38:14.245 : source : tvOS 16 contains 114 new videos (total 114)
2023-05-23 19:38:14.246 : source : From Joshua Michaels & Hal Bergman contains 20 new videos (total 134)
2023-05-23 19:38:14.246 : source : Robin Fourcade's drone shots contains 18 new videos (total 152)
2023-05-23 19:38:14.247 : No coordinates yet, delaying a bit...
2023-05-23 19:38:14.342 : avInit .saver (-1440.0, 0.0, 1440.0, 900.0) p: false o: false
2023-05-23 19:38:14.343 : Best match appearance : Optional(__C.NSAppearanceName(_rawValue: NSAppearanceNameAqua))
2023-05-23 19:38:14.343 : Effective Appearence : <NSCompositeAppearance: 0x7fd24c807da0>
2023-05-23 19:38:14.344 : <AerialView: 0x7fd24bff7900> AerialView setup init (V3.2.5) preview: false
2023-05-23 19:38:14.344 : Running Version 13.3.1 (a) (Build 22E772610a)
2023-05-23 19:38:14.345 : count : 29
2023-05-23 19:38:14.345 : comp_GMT110_112NC_364D_1054_AURORA_ANTARTICA__COMP_FINAL_v34_PS_SDR_20181107_SDR_2K_HEVC.mov
2023-05-23 19:38:14.346 : comp_A083_C002_1130KZ_v04_SDR_PS_FINAL_20180725_SDR_2K_HEVC.mov
2023-05-23 19:38:14.346 : comp_DB_D001_C001_PSNK_v06_SDR_PS_20180824_SDR_2K_HEVC.mov
2023-05-23 19:38:14.346 : comp_H012_C009_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:14.347 : video_inspire_oregon_lone-ranch-beach_00021.1080-sdr.mov
2023-05-23 19:38:14.347 : comp_C004_C003_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:14.348 : comp_A105_C002_v06_SDR_FINAL_25062018_SDR_2K_HEVC.mov
2023-05-23 19:38:14.348 : comp_A006_C003_1219EE_CC_v01_SDR_PS_FINAL_20180709_SDR_2K_HEVC.mov
2023-05-23 19:38:14.349 : Y009_C015_SDR_2K_HEVC.mov
2023-05-23 19:38:14.349 : comp_H004_C007_PS_v02_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:14.350 : comp_LA_A005_C009_PSNK_ALT_v09_SDR_PS_201809134_SDR_2K_HEVC.mov
2023-05-23 19:38:14.350 : comp_GMT307_136NC_134K_8277_NY_NIGHT_01_v25_SDR_PS_20180907_SDR_2K_HEVC.mov
2023-05-23 19:38:14.350 : comp_A009_C001_010181A_v09_SDR_PS_FINAL_20180725_SDR_2K_HEVC.mov
2023-05-23 19:38:14.351 : comp_GMT329_113NC_396B_1105_CHINA_v04_SDR_FINAL_20180706_F900F2700_SDR_2K_HEVC.mov
2023-05-23 19:38:14.351 : video_inspire_california_santa-barbara_050.1080-sdr.mov
2023-05-23 19:38:14.352 : comp_DB_D008_C010_PSNK_v21_SDR_PS_20180914_F0F16157_SDR_2K_HEVC.mov
2023-05-23 19:38:14.352 : comp_N013_C004_PS_v01_SDR_PS_20180925_F1970F7193_SDR_2K_HEVC.mov
2023-05-23 19:38:14.353 : MEX_A006_C008_SDR_20190923_SDR_2K_HEVC.mov
2023-05-23 19:38:14.353 : comp_LA_A008_C004_ALTB_ED_FROM_FLAME_RETIME_v46_SDR_PS_20180917_SDR_2K_HEVC.mov
2023-05-23 19:38:14.354 : comp_H005_C012_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:14.354 : comp_A351_C001_v06_SDR_PS_20180725_SDR_2K_HEVC.mov
2023-05-23 19:38:14.354 : video_inspire_uk_london_hampstead-heath_003.1080-sdr.mov
2023-05-23 19:38:14.355 : comp_L004_C011_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:14.355 : video_inspire_oregon_lone-ranch-beach_00001.1080-sdr.mov
2023-05-23 19:38:14.356 : S003_C020_SDR_2K_HEVC.mov
2023-05-23 19:38:14.356 : comp_HK_B005_C011_PSNK_v16_SDR_PS_20180914_SDR_2K_HEVC.mov
2023-05-23 19:38:14.357 : Manifest opened, going to parsing
2023-05-23 19:38:14.357 : Parsed From Joshua Michaels & Hal Bergman
2023-05-23 19:38:14.358 : Manifest opened, going to parsing
2023-05-23 19:38:14.358 : Parsed Robin Fourcade's drone shots
2023-05-23 19:38:14.358 : Real screen count : 2
2023-05-23 19:38:14.359 : Using : Optional([id=69734406, width=1440, height=900, bottomLeftFrame=(-1440.0, 0.0, 1440.0, 900.0), topRightCorner=(0.0, 900.0), isMain=false, backingScaleFactor=2.0])
2023-05-23 19:38:14.359 : <AerialView: 0x7fd24bff7900> setting up player layer with bounds/frame: (0.0, 0.0, 1440.0, 900.0) / (0.0, 0.0, 1440.0, 900.0)
2023-05-23 19:38:14.360 : allDisplays
2023-05-23 19:38:14.360 : Using dot workaround for video driver corruption
2023-05-23 19:38:14.361 : No coordinates yet, delaying a bit...
2023-05-23 19:38:14.788 : <AerialView: 0x7fd24cc34750> pnv
2023-05-23 19:38:14.789 : <AerialView: 0x7fd24cc34750> Setting player for all player layers in []
2023-05-23 19:38:14.789 : PrefsTime : locationService
2023-05-23 19:38:14.790 : No location available, failing timeMode
2023-05-23 19:38:14.790 : remaining in playlist : 0 needOrientedVideo : false
2023-05-23 19:38:14.791 : generate playlist (isVertical: false
2023-05-23 19:38:14.791 : Playlist raw count: 152 raw cached count 29 isRestricted: false restrictedTo: 
2023-05-23 19:38:14.792 : Final count : 29
2023-05-23 19:38:14.792 : initvideocache
2023-05-23 19:38:15.053 : cached video file with length: 147535386
2023-05-23 19:38:15.053 : AssetLoaderDelegate deinit
2023-05-23 19:38:15.054 : <AerialView: 0x7fd24cc34750> playing video (OFFLINE MODE) : file:///Users/suffo_b/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Application%20Support/Aerial/Cache/comp_C004_C003_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov
2023-05-23 19:38:15.054 : <AerialView: 0x7fd24cc34750> observing current item <AVPlayerItem: 0x7fd24cb9abe0, asset = <AVURLAsset: 0x7fd24cb8fee0, URL = file:///Users/suffo_b/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Application%20Support/Aerial/Cache/comp_C004_C003_PS_v01_SDR_PS_20180925_SDR_2K_HEVC.mov>>
2023-05-23 19:38:15.055 : <AerialView: 0x7fd24cc34750> observeValue Optional("readyForDisplay") false
2023-05-23 19:38:15.079 : <AerialView: 0x7fd24bff7900> pnv
2023-05-23 19:38:15.079 : <AerialView: 0x7fd24bff7900> Setting player for all player layers in []
2023-05-23 19:38:15.080 : PrefsTime : locationService
2023-05-23 19:38:15.080 : > Fill or roll cache
2023-05-23 19:38:15.081 : No location available, failing timeMode
2023-05-23 19:38:15.081 : remaining in playlist : 28 needOrientedVideo : false
2023-05-23 19:38:15.082 : initvideocache
2023-05-23 19:38:15.082 : Looking for hidden videos to delete...
2023-05-23 19:38:15.083 : Looking for outdated videos that aren't in rotation (candidates : 6
2023-05-23 19:38:15.083 : Looking for outdated videos that may still be in rotation (candidates : 6
2023-05-23 19:38:15.083 : Removing outdated video that was in rotation Sheikh Zayed Road
2023-05-23 19:38:15.084 : Uncached videos in rotation : 123
2023-05-23 19:38:15.084 : Queuing video : Jacks
2023-05-23 19:38:15.085 : Video queued Sea
2023-05-23 19:38:15.085 : Starting download for Sea
2023-05-23 19:38:15.087 : cached video file with length: 218650758
2023-05-23 19:38:15.088 : AssetLoaderDelegate deinit
2023-05-23 19:38:15.088 : <AerialView: 0x7fd24bff7900> playing video (OFFLINE MODE) : file:///Users/suffo_b/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Application%20Support/Aerial/Cache/robinfrcd_nusfjord_4k.m4v
2023-05-23 19:38:15.089 : <AerialView: 0x7fd24cc34750> startAnimation frame (0.0, 0.0, 1920.0, 1080.0) bounds (0.0, 0.0, 1920.0, 1080.0)
2023-05-23 19:38:15.089 : <AerialView: 0x7fd24bff7900> observing current item <AVPlayerItem: 0x7fd24cc73030, asset = <AVURLAsset: 0x7fd24cc72f30, URL = file:///Users/suffo_b/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Application%20Support/Aerial/Cache/robinfrcd_nusfjord_4k.m4v>>
2023-05-23 19:38:15.090 : <AerialView: 0x7fd24bff7900> observeValue Optional("readyForDisplay") false
2023-05-23 19:38:15.091 : > Fill or roll cache
2023-05-23 19:38:15.091 : Uncached videos in rotation : 124
2023-05-23 19:38:15.091 : Queuing video : Avalon
2023-05-23 19:38:15.092 : Video queued California
2023-05-23 19:38:15.093 : Starting download for content information
2023-05-23 19:38:15.272 : Starting 1 streams with 89508323 each, for content length of 89508323
2023-05-23 19:38:15.273 : last stream range: {0, 89508323}
2023-05-23 19:38:15.273 : Starting download for range bytes=0-89508323
2023-05-23 19:38:15.391 : <AerialView: 0x7fd24cc34750> observeValue Optional("readyForDisplay") true
2023-05-23 19:38:15.393 : start playback: (0.0, 0.0, 1920.0, 1080.0) (0.0, 0.0, 1920.0, 1080.0) rate: 1.0
2023-05-23 19:38:15.398 : <AerialView: 0x7fd24bff7900> observeValue Optional("readyForDisplay") true
2023-05-23 19:38:15.399 : start playback: (-1440.0, 0.0, 1440.0, 900.0) (0.0, 0.0, 1440.0, 900.0) rate: 1.0
2023-05-23 19:38:16.143 : Couldn't retrieve your location: The operation couldn’t be completed. (kCLErrorDomain error 1.), using latest cached coordinates instead
2023-05-23 19:38:16.144 : Location found 51.461928422503476 -2.587511434976308
2023-05-23 19:38:19.266 : ############ willStop
2023-05-23 19:38:19.269 : <AerialView: 0x7fd24bff7900> stopAnimation
2023-05-23 19:38:19.271 : <AerialView: 0x7fd24bff7900> observeValue Optional("readyForDisplay") false
2023-05-23 19:38:19.273 : ############ willStop
2023-05-23 19:38:19.274 : <AerialView: 0x7fd24cc34750> stopAnimation
2023-05-23 19:38:19.274 : <AerialView: 0x7fd24cc34750> observeValue Optional("readyForDisplay") false
2023-05-23 19:38:19.478 : <AerialView: 0x7fd24cc34750> stopAnimation
```

**Comment by glouel** [comment]
@bensuffolk please see this issue, there is a bug in 13.3.X for multiple monitors : https://github.com/JohnCoates/Aerial/issues/1286

This is partially fixed in 13.4 when you have 2 monitors.
